### PR TITLE
Completed HW1.

### PR DESCRIPTION
**Pull request by sbrennan98**
Edited Program.cs to contain the source code for HW1.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4829267&bid=16045948)

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,43 @@ class Program
...
+
+            BMI = weight / hSquared;
+
+            return BMI;
+        }
+        static void HW1()
+        {
+            //Variable Declarations
+            string name, surname;
+            int age, height; //height in cm
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Declare 'height' in a separate statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4829267)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            hSquared *= hSquared;
+
+            BMI = weight / hSquared;
+
+            return BMI;
+        }
+        static void HW1()
+        {
+            //Variable Declarations
+            string name, surname;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Declare 'surname' in a separate statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4829267)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,43 @@ class Program
...
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Start program
+            Console.WriteLine("Hello World! Welcome to the first C# 2020 Bootcamp Homework.");
+            Program.HW1();
+            Console.WriteLine("Okay, Person #2:");
+            Program.HW1();
+        }
+        static double GetBMI(double weight, int height) //calculates the BMI by receiving weight and height as parameters and dividing weight by height squared (kg/m^2)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Rename method 'GetBMI' to match pascal case naming rules, consider using 'GetBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4829267)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,43 @@ class Program
...
+        {
+            //Function Declarations
+            double BMI, hSquared;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Declare 'hSquared' in a separate statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4829267)

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            name = Console.ReadLine();
+            Console.Write("Hello, " + name + '!' + " I just need a little extra information from you.\nSurname: ");
+            surname = Console.ReadLine();
+            Console.Write("Age: ");
+            age = Convert.ToInt32(Console.ReadLine()); //Console.ReadLine returns a string, so you must convert it to the appropriate type.
+            Console.Write("Weight(in kg): ");
+            weight = Convert.ToDouble(Console.ReadLine());
+            Console.Write("Height(in cm): ");
+            height = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("\n" + name + ' ' + surname + " is " + age + " years old. Their weight is " + weight + "kg, and their height is " + height + "cm tall. This would make their BMI " + GetBMI(weight, height) + '.'); //bug @ Convert.ToString
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Split this 250 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4829267)

**Comment by Almantask** [comment]
Almost great, just please take a look at Codacy suggestions and make changes accordingly. I agree with Codacy.

**Comment by Almantask** [comment]
Please close this PR and make a new one. You are merging to master. You should merge to homework branch.

File: Src/BootCamp.Chapter/Program.cs (hunk 2 above)
Comment (Alumniminium) [reply Src/BootCamp.Chapter/Program.cs]: I don't agree with this issue.

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (Alumniminium) [reply Src/BootCamp.Chapter/Program.cs]: I don't agree with this issue.

File: Src/BootCamp.Chapter/Program.cs (hunk 3 above)
Comment (Alumniminium) [reply Src/BootCamp.Chapter/Program.cs]: I don't agree with this issue.

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (caveofjulian) [reply Src/BootCamp.Chapter/Program.cs]: Looks good.

**Pull request closed without merging**